### ISSUE TITLE
Issue with Ctrl+R: "fzf-history-widget:cd:3: too many arguments"

### ISSUE THREAD

**Issue by kazuuchikata** [junegunn/fzf#1441]
- Category
    - [ ] fzf binary
    - [ ] fzf-tmux script
    - [x] Key bindings
    - [ ] Completion
    - [ ] Vim
    - [ ] Neovim
    - [ ] Etc.
- OS
    - [ ] Linux
    - [x] Mac OS X
    - [ ] Windows
    - [ ] Windows Subsystem for Linux
    - [ ] Etc.
- Shell
    - [ ] bash
    - [x] zsh
    - [ ] fish

When I to Ctrl+R, it gives me an error: `fzf-history-widget:cd:3: too many arguments`
Ctrl+T seems to work just fine. 

I tried reinstalling fzf but the issue still persists. I installed manually and confirmed that my .zshrc file is configured correctly. Thoughts?

**Comment by i-am-eva** [comment]
I don't think this will be it but, try moving your `$HISTFILE` temporarily, does that fix the issue? What is your $HISTSIZE and $SAVEHIST values?
